build.sbt

```diff
@@ -4,6 +4,7 @@ description := "SBT S3 Resolver Plugin"
 
 scalacOptions := Seq(
   "-encoding", "UTF-8",
+  "-target:jvm-1.8",
```

Comment:
needed because of this error
```
[error] /home/joao/git/fm-sbt-s3-resolver/src/main/scala/fm/sbt/S3URLHandler.scala:302:33: Static methods in interface require -target:jvm-1.8
[error]       val tmp = S3Client.builder()
```